ELDistanceFunctions.cginc

```diff
@@ -129,6 +129,16 @@ float fBox2Cheap(float2 pos, float2 box)
 	return max(d.x, d.y);
 }
 
+// Makes a wireframe box at p with scale b and e is how much to hollow out.
+float sdBoundingBox( float3 p, float3 b, float e )

```

Comment:
Some code style cleanup wanted on this function.
Also missing the newline after the closing brace.

Reply:
added newline